README.md

```diff
@@ -958,7 +958,7 @@ Click here to expand for more information.
 	
 #### `older_than`
 
-If omitted first 250 received text messages are returned. Otherwise the next 250 received text messages older than the given id are returned.
+If omitted 250 of the latest received text messages are returned. Otherwise the next 250 received text messages older than the given id are returned.
```

Comment:
‘the 250 most recent‘ is clearer I think.

Reply:
ok